types/express-graphql/index.d.ts

```diff
@@ -46,11 +53,36 @@ declare namespace graphqlHTTP {
          */
         pretty?: boolean | null;
 
+        /**
+         * An optional function which will be used to validate instead of default `validate`
+         * from `graphql-js`.
+         */
+        customValidateFn?: (
+            schema: GraphQLSchema,
+            documentAST: DocumentNode,
+            rules?: ReadonlyArray<any>,
+        ) => ReadonlyArray<GraphQLError> | null;
```

Comment:
The `|` has precedence over `=>` so this definition indicates that the return type could be null rather than the function being null (which is what you want based on the flow types).